src/quacc/recipes/emt/defects.py

```diff
@@ -0,0 +1,98 @@
+"""Defect recipes for EMT"""
+from __future__ import annotations
+
+import covalent as ct
+from ase.atoms import Atoms
+from pymatgen.analysis.defects.generators import (
+    AntiSiteGenerator,
+    ChargeInterstitialGenerator,
+    InterstitialGenerator,
+    SubstitutionGenerator,
+    VacancyGenerator,
+    VoronoiInterstitialGenerator,
+)
+
+from quacc.recipes.emt.core import relax_job, static_job
+from quacc.schemas.ase import OptSchema, RunSchema
+from quacc.util.defects import make_defects_from_bulk
+
+
+def bulk_to_defects_flow(
+    atoms: Atoms | dict,
+    defectgen: (
+        AntiSiteGenerator
+        | ChargeInterstitialGenerator
+        | InterstitialGenerator
+        | SubstitutionGenerator
+        | VacancyGenerator
+        | VoronoiInterstitialGenerator
+    ) = VacancyGenerator,  # NOTE: I added a default. VacancyGenerator seemed like it might be the most popular.
+    charge_state: int = 0,  # NOTE: I changed this from int | None = None because None does not work
+    make_defects_kwargs: dict | None = None,
+    defect_relax: ct.electron | None = relax_job,
+    defect_static: ct.electron | None = static_job,
+    defect_relax_kwargs: dict | None = None,
+    defect_static_kwargs: dict | None = None,
+) -> list[RunSchema | OptSchema]:
+    """
+    Workflow consisting of:
+
+    1. Defect generation
+
+    2. Defect relaxations
```

Comment:
I have made the relaxation step required because there are very few conceivable scenarios where someone would want to make the defect and do only a static calculation.